ref/src/yppm_core.F90

```diff
@@ -570,7 +570,7 @@ subroutine print_2d_variable(name, data)
 
     ! Note: Assumed shape array sections always start with index=1 for all dimensions
     !       So we don't have to know start/end indices here
-    write(*,'(A5, A15,5E20.6)') "TEST ", name, minval(DBLE(data)), maxval(DBLE(data)), DBLE(data(1,1)), &
+    write(*,'(A5, A15,5ES20.6)') "TEST ", name, minval(DBLE(data)), maxval(DBLE(data)), DBLE(data(1,1)), &
```

Comment:
@robgonzalezpita - Could you please change `5ES20.6` to `5ES20.5` to preserve the current number of digits?

Reply:
@christopherwharrop-noaa Sure can!